src/components/Input.react.js

```diff
@@ -162,19 +162,30 @@ Input.propTypes = {
     /**
      * This attribute indicates whether the value of the control can be automatically completed by the browser.
      */
-    autocomplete: PropTypes.string,
+    autoComplete: PropTypes.string,
```

Comment:
yes, given the complexity of autoComplete, I think `string` is a good compromise for now